src/components/messenger/list/group-management/container.tsx

```diff
@@ -78,6 +79,7 @@ export class Container extends React.Component<Properties> {
         profileImage: currentUser?.profileSummary.profileImage,
         matrixId: currentUser?.matrixId,
         isOnline: currentUser?.isOnline,
+        primaryZID: getUserHandle(currentUser?.primaryZID, currentUser?.wallets),
```

Comment:
We probably should have named this `userHandle` or something similar as it could be the wallet address. Maybe something to discuss when we're all on a call

Reply:
I think it should just be `getPrimaryZID(user: User)`/ `getDisplayZID(user)`. 

Something similar to (in `src/lib/user`):
<img width="686" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/9b100234-51ac-471f-9ccf-eaecb2a84483">